frontend/src/pages/Clusters/ClusterScheduling.js

```diff
@@ -0,0 +1,185 @@
+//
+// http://aws.amazon.com/apache2.0/
+//
+// or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+// OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+// limitations under the License.
+import React from 'react';
+
+import { useState, getState, setState, clearState } from '../../store'
+import { QueueStatus, CancelJob } from '../../model'
+import { clusterDefaultUser } from '../../util'
+import { useCollection } from '@awsui/collection-hooks';
+
+import { useTheme } from '@mui/material/styles';
+
+// UI Elements
+import {
+  Button,
+  Pagination,
+  Table,
+  TextFilter
+} from "@awsui/components-react";
+
+// Icons
+import CancelIcon from '@mui/icons-material/Cancel';
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline';
+import CircularProgress from '@mui/material/CircularProgress';
+
+// Components
+import EmptyState from '../../components/EmptyState';
+
+
+function refreshQueues(callback) {
+  const clusterName = getState(['app', 'clusters', 'selected']);
+  if(clusterName){
+    const clusterPath = ['clusters', 'index', clusterName];
+    const cluster = getState(clusterPath);
+    let user = clusterDefaultUser(cluster);
+    const headNode = getState([...clusterPath, 'headNode']);
+    QueueStatus(clusterName, headNode.instanceId, user, callback);
+  }
+}
+
+function Status(props) {
+  const theme = useTheme();
+  const aligned = (icon, text, color) => <div style={{
+    color: color || 'black',
+    display: 'flex',
+    alignItems: 'center',
+    flexWrap: 'wrap',
+  }}>
+    {icon}
+    <span style={{display: 'inline-block', paddingLeft: '10px'}}> {text}</span>
+  </div>
+  const statusMap = {"CANCELLED": aligned(<CancelIcon />, props.status, theme.palette.error.main),
+      "CONFIGURING": aligned(<CircularProgress size={15} color='info' />, props.status, theme.palette.error.main),
+    "RUNNING": aligned(<CheckCircleOutlineIcon />, props.status, theme.palette.success.main),};
+  return props.status in statusMap ? statusMap[props.status] : <span>{props.status}</span>;
+}
+
+function JobActions({job}) {
+  let pendingPath = ['app', 'clusters', 'queue', 'action', job.job_id, 'pending'];
+  const pending = useState(pendingPath);
+
+  const cancelJob = (jobId) => {
+    const clusterName = getState(['app', 'clusters', 'selected']);
+    const clusterPath = ['clusters', 'index', clusterName];
+    const cluster = getState(clusterPath);
+    let user = clusterDefaultUser(cluster);
+    const headNode = getState([...clusterPath, 'headNode']);
+    setState(pendingPath, true);
+    CancelJob(headNode.instanceId, user, jobId, () => refreshQueues(() => clearState(pendingPath)))
+  }
+
+  return (
+    <div>
+      {job.job_state !== "COMPLETED" &&
+      <div>
+        <Button loading={pending} onClick={() => {cancelJob(job.job_id)}}>Stop</Button>
```

Comment:
Button should be named `Cancel`. `Stop` raises the idea of being able to start the job back up

Reply:
done.